app/src/components/Leverager/index.js

```diff
@@ -85,115 +377,107 @@ class Leverager extends PureComponent {
+    if (key === 'ArrowUp') {
       this.setState(
         prevState => ({ priceFloor: prevState.priceFloor + 1 }),
         () => {
-          this.calculateRatioAndLeverage(this.state.priceFloor);
+          this.calculateRatioAndLeverage(this.state.priceFloor)
         }
-      );
+      )
     }
-  };
-  render() {
+  }
+  render () {
     return (
-      <div className="box leverager-box">
-        <div className="leverager-box-header">
-          <div className="leverager-inputs">
-            <div className="input-container">
-              <div className="input-label">Depth:</div>
+      <div className='box leverager-box'>
+        <div className='leverager-box-header'>
+          <div className='leverager-inputs'>
+            <div className='input-container'>
+              <div className='input-label'>Depth:</div>
               <input
-                className="left-padded-input"
+                className='left-padded-input'
                 value={this.state.depth}
-                type="text"
-                placeholder="3"
+                type='text'
+                placeholder='3'
                 onChange={this.setDepth}
                 onKeyDown={this.handleKeyDownDepth}
               />
             </div>
-            <div className="input-container">
-              <div className="input-label">Price Floor:</div>
-              <span
-                style={{ opacity: !!this.state.priceFloor ? "1" : "0.4" }}
-                className="dollar-sign"
-              >
+            <div className='input-container'>
+              <div className='input-label'>Price Floor:</div>
+              <span style={{ opacity: this.state.priceFloor ? '1' : '0.4' }} className='dollar-sign'>
                 $
               </span>
               <input
                 value={this.state.priceFloor}
-                type="text"
-                placeholder="600"
+                type='text'
+                placeholder='600'
                 onChange={this.handleFloorChange}
                 onKeyDown={this.handleKeyDownFloor}
               />
-            </div>{" "}
-            <div className="input-container">
-              <div className="input-label">Amount of ETH:</div>
+            </div>{' '}
+            <div className='input-container'>
+              <div className='input-label'>Amount of ETH:</div>
               <input
-                className="left-padded-input"
+                className='left-padded-input'
                 value={this.state.AmtETH}
-                type="text"
-                placeholder="10"
+                type='text'
+                placeholder='10'
                 onChange={this.handleETHChange}
               />
             </div>
           </div>
-          <div
-            onClick={this.createPosition}
-            style={{ marginTop: "1.5rem", textAlign: "center" }}
-          >
+          <div onClick={this.createPosition} style={{ marginTop: '1.5rem', textAlign: 'center' }}>
```

Comment:
It wasn't immediately obvious to me that this part was clickable in the UI. We need to make it more "button like" imo